packages/stream_chat_flutter/lib/src/channel_header.dart

```diff
@@ -111,26 +128,38 @@ class ChannelHeader extends StatelessWidget implements PreferredSizeWidget {
           child: AppBar(
             brightness: Theme.of(context).brightness,
             elevation: 1,
-            leading: showBackButton
-                ? StreamBackButton(
-                    onPressed: onBackPressed,
-                    showUnreads: true,
-                  )
-                : SizedBox(),
+            leading: leading ??
```

Comment:
Consider extracting to a variable at the top of the build method, keeps the tree clean 😄